src/annotator/plugin/test/pdf-test.js

```diff
@@ -215,4 +215,101 @@ describe('annotator/plugin/pdf', () => {
       assert.notCalled(fakeAnnotator.anchor);
     });
   });
+
+  describe('#contentContainer', () => {
+    let container;
+    afterEach(() => {
+      container?.remove();
+    });
+
+    it('returns main PDF viewer content element', () => {
+      container = document.createElement('div');
+      container.id = 'viewerContainer';
+      document.body.appendChild(container);
+
+      pdfPlugin = createPDFPlugin();
+
+      assert.equal(pdfPlugin.contentContainer(), container);
+    });
+  });
+
+  describe('#fitSideBySide', () => {
```

Comment:
The tests for `PDF#fitSideBySide` were moved and adapted from `pdf-sidebar-test.js`.